Makefile

```diff
@@ -67,6 +67,12 @@ create-nodeport:
 bump-kubevirtci:
 	./hack/bump-kubevirtci.sh
 
+whitespace:
```

Comment:
I find it redundant to use third party github action for the whutespace and on the same time to implemet the logic by ourselves. I would keep just the github action. 
It would be even better if we have an action verifying `make fmt` was invoked. Comparing the existing code and the code after `make fmt`.

Reply:
we dont implement the logic by ourself, the github calls the target of make file

we still need a way to easily fix the whitespace (this is `make whitespace`)
github actions runs the check `make whitespace-check`
the github shows if there is an error, and then you can run locally easily the `make whitespace` in order to fix it easier

about fmt / vet it can be done later, not related to this PR

Reply:
you're right. I missed that, thought we are using a third party action.

Regardung the `fmt`- youre writing - `Please remove trailing whitespaces using `make fmt`:' did you mean `make whitespace`?
If `make fmt` does the job, but `make whitespace` is needed?

Reply:
should be `make whitespace` (which was what i used as well in this PR to fix the whitespaces)
there might be a way also with `make fmt` need to check

Reply:
recall that i did try `make fmt` (and also retried now)
it doesn't fix it, will fix the text string